addons/mail/static/src/components/message/message.xml

```diff
@@ -147,8 +147,10 @@
                             </t>
                             <t t-if="messageView.message.trackingValues.length > 0">
                                 <ul class="o_Message_trackingValues mt-2">
-                                    <t t-foreach="messageView.message.trackingValues" t-as="value" t-key="value.id">
-                                        <TrackingValue value="value"/>
+                                    <t t-foreach="messageView.message.trackingValues" t-as="trackingValue" t-key="trackingValue.id">
```

Comment:
missing view model for tracking component, should we just introduce it later @alexkuhn?

Reply:
The view model could be introduced after this PR is merged.